src/test/java/io/appform/dropwizard/actors/connectivity/actor/NamespacedQueuesTest.java

```diff
@@ -177,6 +244,24 @@ private Response sendRequest(String endpoint) {
         return null;
     }
 
+    private Response sendPostRequest(String endpoint, String body) {
+        OkHttpClient client = new OkHttpClient();
+        String credential = Credentials.basic(config.getUserName(), config.getPassword());
+        Request request = new Request.Builder()
+                .url("http://" + config.getBrokers().get(0).getHost() + ":" + mappedManagementPort + endpoint)
+                .header("Accept", "application/json")
+                .header("Authorization", credential)
+                .post(RequestBody.create(body.getBytes(StandardCharsets.UTF_8)))
+                .build();
+
+        try {
+            return client.newCall(request).execute();
+        } catch (IOException e) {
+            log.error("Error while making API call to RabbitMQ");
+        }
+        return null;
```

Comment:
When would code reach this statement ?

Reply:
When rmq API call fails.

Reply:
For cases where API call to RMQ fails returning null and added assertion of not null response in the test case.